src/devSVG.cpp

```diff
@@ -241,55 +290,59 @@ void svg_clip(double x0, double x1, double y0, double y1, pDevDesc dd) {
 }
 
 void svg_new_page(const pGEcontext gc, pDevDesc dd) {
+BEGIN_RCPP
```

Comment:
Oh yeah, good point.  This code probably would've have been better as pure C, but we might as go fully towards C++, since there are no performance implications here.